cognite/client/data_classes/data_modeling/containers.py

```diff
@@ -140,6 +140,7 @@ def __init__(
         used_for: Literal["node", "edge", "all"] = "node",
         constraints: Optional[dict[str, Constraint]] = None,
         indexes: Optional[dict[str, Index]] = None,
+        **_: dict,
```

Comment:
This type annotation implies all kwargs are actual dicts:
```suggestion
        **_: Any,
```

Reply:
I did a search and see that this error has been made all over --> feel free to ignore it here as well.